script/room.js

```diff
@@ -886,6 +880,15 @@ var Room = {
 		}
 	},
 	
+	location_call: function(i) {
+		var child = $(this);
+		var cName = child.children('.row_key').text();
+		if(cName < lk) {
+			curPrev = child.attr('id');
```

Comment:
`curPrev` is undefined here, as it was previously coming from the closure scope of the inline function.